src/ext/VSExtension/wixlib/VsixPackage.wxs

```diff
@@ -22,13 +22,19 @@
             </RegistrySearch>
         </Property>
 
-        <!-- VS2015 is the latest VSIX at this point in time, so search for that by default. -->
         <Property Id="VS2015_VSIX_INSTALLER_PATH" Secure="yes">
             <RegistrySearch Id="WixVS2015EnvDir" Root="HKLM" Key="SOFTWARE\Microsoft\VisualStudio\14.0\Setup\VS" Name="EnvironmentDirectory" Type="directory">
                 <FileSearch Id="WixVS2015VsixInstaller" Name="VSIXInstaller.exe" />
             </RegistrySearch>
         </Property>
 
+        <!-- Currently, VS2017 contains the latest VSIX installer, so search it by default. -->
+        <Property Id="VS2017_VSIX_INSTALLER_PATH" Secure="yes">
+            <DirectorySearchRef Id="VS2017EnvironmentDirectorySearch" Parent="VS2017DirectorySearch" Path="Common7\IDE">
+                <FileSearch Id="WixVS2017VsixInstallerPath" Name="VSIXInstaller.exe" />
```

Comment:
We recommend using the VSIXInstaller in the search engine root. We update that one with the installer so it's always newer and may eventually remove the one installed with VS.

Since I set the VS2017 property to the latest version, this would be a fine backup, but the installer party is already set by the custom actions.

The real problem is that if a VSIX triggers an MSI install, the install will fail. There's a WIP on good you should read if you want to improve this feature.